doc/contributor/howto-guide-setup-cmake-environment.md

```diff
@@ -43,26 +43,42 @@ you use Ninja for this build because it is substantially faster than Make for
 these use-cases. Note that if Ninja is not installed in your workstation you may
 need to remove the `-GNinja` flag:
 
-```console
+```shell
 cmake -Hsuper -Bcmake-out/si \
     -DGOOGLE_CLOUD_CPP_EXTERNAL_PREFIX=$HOME/local-cpp -GNinja
 ```
 
 Install the dependencies:
 
-```console
-cmake --build cmake-out/si --target project-dependencies -- -j $(nproc)
+```shell
+cmake --build cmake-out/si --target project-dependencies
+```
+
+If you are **not** using Ninja in your build you may need to specify a `-j`
```

Comment:
(optional), this is a lot of lines talking about `-j`. You might consider saying only

```
NOTE: Ninja automatically parallelizes your build based on the number of cores on your machine. If you are **not** using Ninja, you may need to specify `-j <N>` if you want to parallelize your build.
```
and that's it. I think there's no need to mention that we're omitting `-j` because it's clear from above that this document *is* using Ninja.

Reply:
Done.